src/wifi.c

```diff
@@ -0,0 +1,84 @@
+#include <freertos/FreeRTOS.h>
+#include <freertos/event_groups.h>
+#include <esp_event_loop.h>
+#include <esp_wifi.h>
+#include <string.h>
+#include <stdlib.h>
+#include <nvs_flash.h>
+
+#include "wifi.h"
+
+#define CONNECTED_BIT BIT0
```

Comment:
does this value ever change? if not, why is it an extra define?

Reply:
To improve readability. Using `BIT0` would be like using magic numbers